index.bs

```diff
@@ -670,6 +686,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
     </div>
     </li>
 
+    <li><dfn>Focus distance</dfn> is a numeric camera setting that controls the focus distance of the lens.  The setting usually represents distance in millimeters to the optimal focus distance.</li>
```

Comment:
@jan-ivar, fair points, especially about persisting settings across different cameras.

@yellowdoge, should I drop 'usually' and instead state that focus distance is in meters? Sounds like the consensus is to lock down the unit to meters, and if we're going to do that then it probably does more harm than good to say 'usually' since that leaves room for interpretation.